### DISCUSSION TITLE
Support for ScyllaDB/DynamoDB/Cassandra

### DISCUSSION THREAD

**Discussion by aprilthepink** [mikro-orm/mikro-orm#3954] [General]
**Is your feature request related to a problem? Please describe.**
I found this cool tool, cause I wanted to give my users options to use different database softwares in their instance of my software, and as I currently only use typescript with a rather bad orm, I concidered doing the switch to this orm.  But we decided with the users that its more sensible to offer more than one DB software to be used.  And there this orm seemed to come in handy, but we quickly discovered that it has no support for our planned scylla support. And keeping to look for another orm which has support for that seemed non-sensical as it already has support for mongo and postgres, which we plan on supporting.

**Describe the solution you'd like**
Support for Scylla either by a DynamoDB or Cassandra driver, as Scylla supports both.

**Describe alternatives you've considered**
Writing the complete driver / orm myself. Which is much work for a 2 person team of a OSP.

**Additional context**
Sorry for not writing much, its 3am ><

**Comment by B4nan** [comment]
I will take a closer look on what does this actually mean, but I don't plan to introduce more drivers in near future, the only ones I am planning are cockroach and mssql. It might be fairly easy to support dynamo, but I have zero experience with that, and I would rather not add more drivers I have no knowledge about.

> Writing the complete driver / orm myself. Which is much work for a 2 person team of a OSP.

Uh, well, I am 1 person :] I am sure you guys can put together an ORM driver, you don't need to write your own ORM for that :] Most of the mongo driver code could be probably reused, and it's not very complex.

I'll be happy to help with this, but I don't think I will be able to drive it. Ideally, this should be born as a 3rd party driver, we can integrate it to the ORM when we see how it works.

**Comment by aprilthepink** [comment]
yea, dynamo is basically mongo but with a few differences. tbh I think its that similar that it could even be made into a "main" driver like mongo once its finished. I know you said smth different, but I think the differences are only api wise and not orm wise

**Reply by B4nan** [reply]
sure, as long as it will be similar enough. i am not saying it will never make it to the ORM monorepo, in the long run i would definitely like to have support for it, but currently its not a priority - but it might be a low hanging fruit in the end.

**Comment by aprilthepink** [comment]
PS: appart from the fact that mongo uses bson and dynamo doesnt
